.gitignore

```diff
@@ -49,7 +49,6 @@ openzeppelin-docs
 contracts/build/
 contracts/README.md
 
-package-lock.json
```

Comment:
I'm not sure why you ignored package-lock.json ...?

Isn't it better to ensure that everyone is locked to the exact same version of development dependencies?

Reply:
Hmm yup, can't remember why I ignored it.